book.toml

```diff
@@ -0,0 +1,22 @@
+[book]
+title = "Programmeringskurs for viderekomne"
+authors = ["Thorben Werner Sjøstrøm Dahl", "Teodor Ande Elstad", "Heidi Mork", "Per Edvard Volla"]
+description = "Innføring i mer avanserte programmeringskonsepter som kan monne i hverdagen"
+language = "nb"
```

Comment:
Er det mulig å sette src = "" ( eller `.`, `/`) for at filene med dag0 og dag1 fortsatt er i rot-mappa? Hadde vært fint å slippe den ekstra folderen men det er kanskje ikke mulig?

Reply:
Jeg tok og prøvde det nå. mdbook kommer med en advarsel om at theme-mappa ikke egentlig er ment å ligge i src, men utover det får den til å lage nettsida. Det som er problematisk, er at den vil kopiere inn alle filene i src-mappa til build-mappa. Så da vil hele repo-innholdet bli publisert, deriblant .git, .github med flere

Reply:
ja, det er jo litt dumt det også. Fant ikke noen åpenbar måte å ignorere filer på heller, så det var issue åpent for å legge til en .ignore-fil eller noe. Så da er det sikkert best å ha det sånn du har det nå.